<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormCreateActivityWizardCodingTests</class>
 <widget class="QWidget" name="FormCreateActivityWizardCodingTests">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>626</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>501</width>
    <height>626</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QToolBox" name="toolBox_codingtest_main">
     <property name="styleSheet">
      <string notr="true">QTabWidget::tab-bar { height: 20px; width: 500px; }</string>
     </property>
     <widget class="QWidget" name="toolBox_codingtest_mainPage1" native="true">
      <attribute name="label">
       <string extracomment="General Name and Description">General Name and Description</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLineEdit" name="lineEdit_codingtest_outputPath"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Console Output File Path</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLineEdit" name="lineEdit_codingtest_errorPath"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Console Input File Path</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Console Error File Path</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLineEdit" name="lineEdit_codingtest_inputpath"/>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="pushButton_codingtest_inputPath">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QPushButton" name="pushButton_codingtest_outputPath">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QPushButton" name="pushButton_codingtest_errorpath">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QTextEdit" name="textEdit_codingTest_description"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLineEdit" name="lineEdit_codingTest_name"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="toolBox_codingtest_mainPage2" native="true">
      <attribute name="label">
       <string>File Names and Paths</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_codingtest_solAdd">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_codingtest_solDel">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="3">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="5">
          <widget class="QTableView" name="tableView_codingtest_solPath"/>
         </item>
         <item row="0" column="0" colspan="5">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Solution Path</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_11">
         <item row="1" column="0" colspan="4">
          <widget class="QTableView" name="tableView_codingtest_subPath"/>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_codingtest_subAdd">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Submission Path</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_codingtest_subDel">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="1" column="0" colspan="4">
          <widget class="QTableView" name="tableView_codingtest_outputPath"/>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButton_codingtest_outputAdd">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Output Path</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButton_codingtest_outputDel">
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_Cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_confirm">
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
